trackintel/visualization/triplegs.py

```diff
@@ -20,36 +20,42 @@ def plot_triplegs(triplegs, out_filename=None, positionfixes=None, staypoints=No
     positionfixes : GeoDataFrame
         If available, some positionfixes that can additionally be plotted.
 
+    staypoints_radius : float, optional
+        The radius in meter with which circles around staypoints should be drawn.
+
     plot_osm : bool
         If this is set to True, it will download an OSM street network and plot 
         below the triplegs.
+
+    axis : matplotlib.pyplot.Artist, optional
+        axis on which to draw the plot
+
     """
```

Comment:
Please add example in docstring (see point 15 in the [docstring guide](https://numpydoc.readthedocs.io/en/latest/format.html#sections)